packages/aetna/assets/styles/layouts/_header.scss

```diff
@@ -13,6 +8,11 @@
   justify-content: space-between;
   max-width: $nav-max-width;
   margin: 0 auto;
+  padding: 3.5rem 1rem 1.5rem;
```

Comment:
move padding to `.header__inside` element so new CTA can stretch full width of body container